q2_composition/_diff_abundance_plots.py

```diff
@@ -0,0 +1,260 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from pathlib import Path
+from collections import OrderedDict
+
+import altair as alt
+import pandas as pd
+import numpy as np
+
+import qiime2
+from q2_composition import DataLoafPackageDirFmt
+
+_html_head = """<html>
+<head>
+<style>
+    body {
+        padding: 20px;
+        font-family: Verdana, sans-serif;
+    }
+    div {
+        width: 60%;
+    }
+</style>
+<meta charset="UTF-8">
+</head>
+<body>
+<p>Click a link to see the differential abundance bar plot for the specified
+   category:
+</p>
+<ul>
+"""
+
+_html_foot = """</ul>
+<div>
+<hr>
+<p>Notes on interpreting plots with taxonomic feature identifiers:</p>
+<ul>
+<li>If taxonomic labels are used to identify features, the feature labels
+(y-axis labels) in each plot represent the most specific named taxonomic level
+associated with that feature.</li>
+<li>Hover over the bars in plots to see the full taxonomic label of each
+feature identifier and information about its differential abundance relative
+to the reference.</li>
+<li>Feature identifiers (y-axis labels) that are followed by an asterisk
+(<code>*</code>) represent instances of a duplicated taxonomic name at the
+level displayed in the feature identifier. The number next to the feature
+identifiers in these cases is used only for unique identification in the
+current figure. It is not taxonomically meaningful, and won't be
+consistent across visualizations.</li>
+</ul>
+</div>
+</body>
+</html>
+"""
```

Comment:
A jinja template is probably the better approach. You can see an example here: 
https://github.com/qiime2/q2-composition/tree/master/q2_composition/_dataloaf_tabulate

Reply:
Agreed. I'll try to swap this out in time for the release.

Reply:
Ok, this is using a jinja template now. Thanks for the suggestion!